sermant-plugins/sermant-router/config/config.yaml

```diff
@@ -1,22 +1,26 @@
 router.plugin:
-  # 是否适配注册插件
+  # whether compatible with the registration plugin
   enabled-registry-plugin-adaptation: false
-  # 是否使用请求信息做路由
+  # Whether to use request information for routing
   use-request-router: false
-  # 使用请求信息做路由时的tags
+  # use the request information as a route tags
   request-tags: []
-  # 需要解析的请求头的tag
+  # the tag of the request header to be resolved
   parse-header-tag: ''
-  # 是否初始化dubbo区域路由，兼容性开关，开启后，会在dubbo的配置缓存中初始化一条以zone匹配的同标签路由
+  # Whether to initialize DUBBO zone routing and whether to switch compatibility toggles，After this feature is enabled,
+  # a route with the same label that matches the zone is initialized in the configuration cache of Dubbo
   enabled-dubbo-zone-router: false
-  # 是否初始化spring cloud区域路由，兼容性开关，开启后，会在spring cloud的配置缓存中初始化一条以zone匹配的同标签路由
+  # Whether to initialize the spring cloud area routing and compatibility switch?
+  # Once enabled, a routing with the same label that matches the zone will be initialized in the spring cloud
+  # configuration cache
```

Comment:
Whether to initialize DUBBO zone routing（compatibility switch）
Whether to initialize the spring cloud zone routing （compatibility switch）